### PR TITLE
feat(Archiving) Add new database table

### PR DESCRIPTION
**Pull request by Vince-janv**
## Description
To keep track of which files are archived/retrieve and when a new table has been created, linked one-to-one with files

### Added

- New table archive

### How to prepare for test

- [ ] ssh to hasta (depending on type of change)
- [ ] install on stage:
```bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_main -t housekeeper -b alembic-add-archive-table -a```

### How to test
- [ ] login to ...
- [ ] do ...

### Expected test outcome
- [ ] check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review
- [ ] code approved by
- [ ] tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: housekeeper/store/models.py [hunk 1]
```diff
@@ -18,8 +18,20 @@
 )
 
 
-class Bundle(Model):
+class Archive(Model):
+    """Information regarding the archival of a file"""
```
Comment (diitaz93) [inline housekeeper/store/models.py:22]:
```suggestion
    """Information regarding the archival of a file."""
```

File: housekeeper/store/models.py [hunk 2]
```diff
@@ -97,7 +107,6 @@ def is_included(self) -> bool:
 
 
 class Tag(Model):
-
     """Represents tags for bundles"""
```
Comment (diitaz93) [inline housekeeper/store/models.py:110]:
```suggestion
    """Represents tags for bundles."""
```

File: housekeeper/store/models.py [hunk 3]
```diff
@@ -36,7 +48,6 @@ class Bundle(Model):
 
 
 class Version(Model):
-
     """Keeps track of versions"""
```
Comment (diitaz93) [inline housekeeper/store/models.py:51]:
```suggestion
    """Keeps track of versions."""
```

File: housekeeper/store/api/handlers/base.py [hunk 4]
```diff
@@ -1,13 +1,14 @@
 """This module defines a BaseHandler class holding different models"""
 
 from typing import Type
-from housekeeper.store.models import Bundle, File, Version, Tag, Model
+from housekeeper.store.models import Archive, Bundle, File, Version, Tag, Model
 from sqlalchemy.orm import Query, Session
 
 
 class BaseHandler:
     """This is a base class holding different models"""
```
Comment (diitaz93) [inline housekeeper/store/api/handlers/base.py:9]:
```suggestion
    """This is a base class holding different models."""
```

**Review by diitaz93 — APPROVED** [review]
👍

**Review by seallard — APPROVED** [review]
Looks good 👍

File: housekeeper/store/models.py [hunk 5]
```diff
@@ -18,8 +18,20 @@
...
 
 
-class Bundle(Model):
+class Archive(Model):
+    """Information regarding the archival of a file"""
+
+    __tablename__ = "archive"
+    archiving_task_id = Column(types.Integer, nullable=False)
+    retrieval_task_id = Column(types.Integer, nullable=True)
+    file_id = Column(ForeignKey("file.id"), nullable=False, primary_key=True)
```
[the remark this answers is not in the document]
Comment (seallard) [reply housekeeper/store/models.py:27]: Never mind, I guess this way enforces a 1-1 relationship between the models.

File: alembic/versions/f1669dd42064_add_archive_table.py [hunk 6]
```diff
@@ -0,0 +1,36 @@
+"""empty message Creates a new table describing archiving and retrieval tasks
```
Comment (henrikstranneheim) [inline alembic/versions/f1669dd42064_add_archive_table.py:1]:
```suggestion
"""Creates a new table describing archiving and retrieval tasks
```

**Review by henrikstranneheim — APPROVED** [review]
👍

**Comment by Vince-janv** [comment]
### Deployed to stage
```
bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_main -t housekeeper -b master -a
Collecting git+https://github.com/Clinical-Genomics/housekeeper@master
  Running command git clone --quiet https://github.com/Clinical-Genomics/housekeeper /home/vincent.janvid/tmp/pip-req-build-2lijfqa9
  Cloning https://github.com/Clinical-Genomics/housekeeper (to revision master) to /home/vincent.janvid/tmp/pip-req-build-2lijfqa9
  Resolved https://github.com/Clinical-Genomics/housekeeper to commit e8e073676d27773869c568f3a7f8977f5b3460c1
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Building wheels for collected packages: housekeeper
  Building wheel for housekeeper (setup.py): started
  Building wheel for housekeeper (setup.py): finished with status 'done'
  Created wheel for housekeeper: filename=housekeeper-4.2.0-py2.py3-none-any.whl size=29478 sha256=3d1a032bc3f1b1800398ac7c43db2b2ef3ae28ebdd8eea2bc5cfb133db8af314
  Stored in directory: /home/vincent.janvid/tmp/pip-ephem-wheel-cache-2vc8gwxv/wheels/1f/b9/8a/3efc5008f9806158878d60f365de4f523db145d5ff1aaf5081
Successfully built housekeeper
Installing collected packages: housekeeper
  Attempting uninstall: housekeeper
    Found existing installation: housekeeper 4.2.0
    Uninstalling housekeeper-4.2.0:
      Successfully uninstalled housekeeper-4.2.0
Successfully installed housekeeper-4.2.0
Collecting git+https://github.com/Clinical-Genomics/housekeeper@master
  Running command git clone --quiet https://github.com/Clinical-Genomics/housekeeper /home/vincent.janvid/tmp/pip-req-build-64hxj6s7
  Cloning https://github.com/Clinical-Genomics/housekeeper (to revision master) to /home/vincent.janvid/tmp/pip-req-build-64hxj6s7
  Resolved https://github.com/Clinical-Genomics/housekeeper to commit e8e073676d27773869c568f3a7f8977f5b3460c1
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Requirement already satisfied: Click<7 in /home/proj/stage/bin/miniconda3/envs/S_main/lib/python3.7/site-packages (from housekeeper==4.2.0) (6.7)
Requirement already satisfied: coloredlogs in /home/proj/stage/bin/miniconda3/envs/S_main/lib/python3.7/site-packages (from housekeeper==4.2.0) (14.3)
Requirement already satisfied: marshmallow in /home/proj/stage/bin/miniconda3/envs/S_main/bin (from housekeeper==4.2.0) (3.18.0)
Requirement already satisfied: pyyaml in /home/proj/stage/bin/miniconda3/envs/S_main/lib/python3.7/site-packages (from housekeeper==4.2.0) (5.4.1)
Requirement already satisfied: SQLAlchemy<1.4 in /home/proj/stage/bin/miniconda3/envs/S_main/lib/python3.7/site-packages (from housekeeper==4.2.0) (1.3.24)
Requirement already satisfied: rich in /home/proj/stage/bin/miniconda3/envs/S_main/bin (from housekeeper==4.2.0) (12.6.0)
Requirement already satisfied: humanfriendly>=7.1 in /home/proj/stage/bin/miniconda3/envs/S_main/bin (from coloredlogs->housekeeper==4.2.0) (10.0)
Requirement already satisfied: packaging>=17.0 in /home/proj/stage/bin/miniconda3/envs/S_main/bin (from marshmallow->housekeeper==4.2.0) (21.3)
Requirement already satisfied: pygments<3.0.0,>=2.6.0 in /home/proj/stage/bin/miniconda3/envs/S_main/bin (from rich->housekeeper==4.2.0) (2.13.0)
Requirement already satisfied: commonmark<0.10.0,>=0.9.0 in /home/proj/stage/bin/miniconda3/envs/S_main/bin (from rich->housekeeper==4.2.0) (0.9.1)
Requirement already satisfied: typing-extensions<5.0,>=4.0.0 in /home/proj/stage/bin/miniconda3/envs/S_main/lib/python3.7/site-packages (from rich->housekeeper==4.2.0) (4.4.0)
Requirement already satisfied: pyparsing!=3.0.5,>=2.0.2 in /home/proj/stage/bin/miniconda3/envs/S_main/bin (from packaging>=17.0->marshmallow->housekeeper==4.2.0) (3.0.9)
INFO  [alembic.runtime.migration] Context impl MySQLImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
INFO  [alembic.runtime.migration] Context impl MySQLImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
Already on 'master'
Already up-to-date.
/home/proj/stage/bin/git/housekeeper
INFO  [alembic.runtime.migration] Context impl MySQLImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
INFO  [alembic.runtime.migration] Running upgrade 9334201fec74 -> f1669dd42064, Creates a new table describing archiving and retrieval tasks
repository is clean
Logging deploy ...
Getting deployer... done.
Getting last commit message and SHA... done.
Getting version of deploy scripts... /home/proj/stage/bin/git/housekeeper
done.
Log deploy... done.
housekeeper, version 4.2.0
```
### Deployed to production
```
bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-prod.sh -e P_main -t housekeeper -b master -a
Collecting git+https://github.com/Clinical-Genomics/housekeeper@master
  Running command git clone --quiet https://github.com/Clinical-Genomics/housekeeper /home/vincent.janvid/tmp/pip-req-build-68ym2bxj
  Cloning https://github.com/Clinical-Genomics/housekeeper (to revision master) to /home/vincent.janvid/tmp/pip-req-build-68ym2bxj
  Resolved https://github.com/Clinical-Genomics/housekeeper to commit e8e073676d27773869c568f3a7f8977f5b3460c1
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Building wheels for collected packages: housekeeper
  Building wheel for housekeeper (setup.py): started
  Building wheel for housekeeper (setup.py): finished with status 'done'
  Created wheel for housekeeper: filename=housekeeper-4.2.0-py2.py3-none-any.whl size=29478 sha256=5f70ce6633f57ccfda42b0829f9d7a6f5feb264514d803c3014786431afab74f
  Stored in directory: /home/vincent.janvid/tmp/pip-ephem-wheel-cache-8vvqgyje/wheels/1f/b9/8a/3efc5008f9806158878d60f365de4f523db145d5ff1aaf5081
Successfully built housekeeper
Installing collected packages: housekeeper
  Attempting uninstall: housekeeper
    Found existing installation: housekeeper 4.1.0
    Uninstalling housekeeper-4.1.0:
      Successfully uninstalled housekeeper-4.1.0
Successfully installed housekeeper-4.2.0
Collecting git+https://github.com/Clinical-Genomics/housekeeper@master
  Running command git clone --quiet https://github.com/Clinical-Genomics/housekeeper /home/vincent.janvid/tmp/pip-req-build-wav_tqez
  Cloning https://github.com/Clinical-Genomics/housekeeper (to revision master) to /home/vincent.janvid/tmp/pip-req-build-wav_tqez
  Resolved https://github.com/Clinical-Genomics/housekeeper to commit e8e073676d27773869c568f3a7f8977f5b3460c1
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Requirement already satisfied: Click<7 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from housekeeper==4.2.0) (6.7)
Requirement already satisfied: coloredlogs in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from housekeeper==4.2.0) (15.0.1)
Requirement already satisfied: marshmallow in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from housekeeper==4.2.0) (3.18.0)
Requirement already satisfied: pyyaml in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from housekeeper==4.2.0) (6.0)
Requirement already satisfied: SQLAlchemy<1.4 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from housekeeper==4.2.0) (1.3.24)
Requirement already satisfied: rich in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from housekeeper==4.2.0) (12.6.0)
Requirement already satisfied: humanfriendly>=9.1 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from coloredlogs->housekeeper==4.2.0) (10.0)
Requirement already satisfied: packaging>=17.0 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from marshmallow->housekeeper==4.2.0) (21.3)
Requirement already satisfied: commonmark<0.10.0,>=0.9.0 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from rich->housekeeper==4.2.0) (0.9.1)
Requirement already satisfied: typing-extensions<5.0,>=4.0.0 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from rich->housekeeper==4.2.0) (4.4.0)
Requirement already satisfied: pygments<3.0.0,>=2.6.0 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from rich->housekeeper==4.2.0) (2.13.0)
Requirement already satisfied: pyparsing!=3.0.5,>=2.0.2 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from packaging>=17.0->marshmallow->housekeeper==4.2.0) (2.4.7)
Already on 'master'
Already up-to-date.
/home/proj/production/bin/git/housekeeper
INFO  [alembic.runtime.migration] Context impl MySQLImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
INFO  [alembic.runtime.migration] Running upgrade 9334201fec74 -> f1669dd42064, Creates a new table describing archiving and retrieval tasks
repository is clean
Logging deploy ...
Getting deployer... done.
Getting last commit message and SHA... done.
Getting version of deploy scripts... /home/proj/production/bin/git/housekeeper
done.
Log deploy... done.
housekeeper, version 4.2.0
```

**Comment by Vince-janv** [comment]
### Deployed to stage
```
bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_main -t housekeeper -b master -a
Collecting git+https://github.com/Clinical-Genomics/housekeeper@master
  Running command git clone --quiet https://github.com/Clinical-Genomics/housekeeper /home/vincent.janvid/tmp/pip-req-build-2lijfqa9
  Cloning https://github.com/Clinical-Genomics/housekeeper (to revision master) to /home/vincent.janvid/tmp/pip-req-build-2lijfqa9
  Resolved https://github.com/Clinical-Genomics/housekeeper to commit e8e073676d27773869c568f3a7f8977f5b3460c1
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Building wheels for collected packages: housekeeper
  Building wheel for housekeeper (setup.py): started
  Building wheel for housekeeper (setup.py): finished with status 'done'
  Created wheel for housekeeper: filename=housekeeper-4.2.0-py2.py3-none-any.whl size=29478 sha256=3d1a032bc3f1b1800398ac7c43db2b2ef3ae28ebdd8eea2bc5cfb133db8af314
  Stored in directory: /home/vincent.janvid/tmp/pip-ephem-wheel-cache-2vc8gwxv/wheels/1f/b9/8a/3efc5008f9806158878d60f365de4f523db145d5ff1aaf5081
Successfully built housekeeper
Installing collected packages: housekeeper
  Attempting uninstall: housekeeper
    Found existing installation: housekeeper 4.2.0
    Uninstalling housekeeper-4.2.0:
      Successfully uninstalled housekeeper-4.2.0
Successfully installed housekeeper-4.2.0
Collecting git+https://github.com/Clinical-Genomics/housekeeper@master
  Running command git clone --quiet https://github.com/Clinical-Genomics/housekeeper /home/vincent.janvid/tmp/pip-req-build-64hxj6s7
  Cloning https://github.com/Clinical-Genomics/housekeeper (to revision master) to /home/vincent.janvid/tmp/pip-req-build-64hxj6s7
  Resolved https://github.com/Clinical-Genomics/housekeeper to commit e8e073676d27773869c568f3a7f8977f5b3460c1
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Requirement already satisfied: Click<7 in /home/proj/stage/bin/miniconda3/envs/S_main/lib/python3.7/site-packages (from housekeeper==4.2.0) (6.7)
Requirement already satisfied: coloredlogs in /home/proj/stage/bin/miniconda3/envs/S_main/lib/python3.7/site-packages (from housekeeper==4.2.0) (14.3)
Requirement already satisfied: marshmallow in /home/proj/stage/bin/miniconda3/envs/S_main/bin (from housekeeper==4.2.0) (3.18.0)
Requirement already satisfied: pyyaml in /home/proj/stage/bin/miniconda3/envs/S_main/lib/python3.7/site-packages (from housekeeper==4.2.0) (5.4.1)
Requirement already satisfied: SQLAlchemy<1.4 in /home/proj/stage/bin/miniconda3/envs/S_main/lib/python3.7/site-packages (from housekeeper==4.2.0) (1.3.24)
Requirement already satisfied: rich in /home/proj/stage/bin/miniconda3/envs/S_main/bin (from housekeeper==4.2.0) (12.6.0)
Requirement already satisfied: humanfriendly>=7.1 in /home/proj/stage/bin/miniconda3/envs/S_main/bin (from coloredlogs->housekeeper==4.2.0) (10.0)
Requirement already satisfied: packaging>=17.0 in /home/proj/stage/bin/miniconda3/envs/S_main/bin (from marshmallow->housekeeper==4.2.0) (21.3)
Requirement already satisfied: pygments<3.0.0,>=2.6.0 in /home/proj/stage/bin/miniconda3/envs/S_main/bin (from rich->housekeeper==4.2.0) (2.13.0)
Requirement already satisfied: commonmark<0.10.0,>=0.9.0 in /home/proj/stage/bin/miniconda3/envs/S_main/bin (from rich->housekeeper==4.2.0) (0.9.1)
Requirement already satisfied: typing-extensions<5.0,>=4.0.0 in /home/proj/stage/bin/miniconda3/envs/S_main/lib/python3.7/site-packages (from rich->housekeeper==4.2.0) (4.4.0)
Requirement already satisfied: pyparsing!=3.0.5,>=2.0.2 in /home/proj/stage/bin/miniconda3/envs/S_main/bin (from packaging>=17.0->marshmallow->housekeeper==4.2.0) (3.0.9)
INFO  [alembic.runtime.migration] Context impl MySQLImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
INFO  [alembic.runtime.migration] Context impl MySQLImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
Already on 'master'
Already up-to-date.
/home/proj/stage/bin/git/housekeeper
INFO  [alembic.runtime.migration] Context impl MySQLImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
INFO  [alembic.runtime.migration] Running upgrade 9334201fec74 -> f1669dd42064, Creates a new table describing archiving and retrieval tasks
repository is clean
Logging deploy ...
Getting deployer... done.
Getting last commit message and SHA... done.
Getting version of deploy scripts... /home/proj/stage/bin/git/housekeeper
done.
Log deploy... done.
housekeeper, version 4.2.0
```
### Deployed to production
```
bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-prod.sh -e P_main -t housekeeper -b master -a
Collecting git+https://github.com/Clinical-Genomics/housekeeper@master
  Running command git clone --quiet https://github.com/Clinical-Genomics/housekeeper /home/vincent.janvid/tmp/pip-req-build-68ym2bxj
  Cloning https://github.com/Clinical-Genomics/housekeeper (to revision master) to /home/vincent.janvid/tmp/pip-req-build-68ym2bxj
  Resolved https://github.com/Clinical-Genomics/housekeeper to commit e8e073676d27773869c568f3a7f8977f5b3460c1
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Building wheels for collected packages: housekeeper
  Building wheel for housekeeper (setup.py): started
  Building wheel for housekeeper (setup.py): finished with status 'done'
  Created wheel for housekeeper: filename=housekeeper-4.2.0-py2.py3-none-any.whl size=29478 sha256=5f70ce6633f57ccfda42b0829f9d7a6f5feb264514d803c3014786431afab74f
  Stored in directory: /home/vincent.janvid/tmp/pip-ephem-wheel-cache-8vvqgyje/wheels/1f/b9/8a/3efc5008f9806158878d60f365de4f523db145d5ff1aaf5081
Successfully built housekeeper
Installing collected packages: housekeeper
  Attempting uninstall: housekeeper
    Found existing installation: housekeeper 4.1.0
    Uninstalling housekeeper-4.1.0:
      Successfully uninstalled housekeeper-4.1.0
Successfully installed housekeeper-4.2.0
Collecting git+https://github.com/Clinical-Genomics/housekeeper@master
  Running command git clone --quiet https://github.com/Clinical-Genomics/housekeeper /home/vincent.janvid/tmp/pip-req-build-wav_tqez
  Cloning https://github.com/Clinical-Genomics/housekeeper (to revision master) to /home/vincent.janvid/tmp/pip-req-build-wav_tqez
  Resolved https://github.com/Clinical-Genomics/housekeeper to commit e8e073676d27773869c568f3a7f8977f5b3460c1
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Requirement already satisfied: Click<7 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from housekeeper==4.2.0) (6.7)
Requirement already satisfied: coloredlogs in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from housekeeper==4.2.0) (15.0.1)
Requirement already satisfied: marshmallow in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from housekeeper==4.2.0) (3.18.0)
Requirement already satisfied: pyyaml in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from housekeeper==4.2.0) (6.0)
Requirement already satisfied: SQLAlchemy<1.4 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from housekeeper==4.2.0) (1.3.24)
Requirement already satisfied: rich in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from housekeeper==4.2.0) (12.6.0)
Requirement already satisfied: humanfriendly>=9.1 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from coloredlogs->housekeeper==4.2.0) (10.0)
Requirement already satisfied: packaging>=17.0 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from marshmallow->housekeeper==4.2.0) (21.3)
Requirement already satisfied: commonmark<0.10.0,>=0.9.0 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from rich->housekeeper==4.2.0) (0.9.1)
Requirement already satisfied: typing-extensions<5.0,>=4.0.0 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from rich->housekeeper==4.2.0) (4.4.0)
Requirement already satisfied: pygments<3.0.0,>=2.6.0 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from rich->housekeeper==4.2.0) (2.13.0)
Requirement already satisfied: pyparsing!=3.0.5,>=2.0.2 in /home/proj/production/bin/miniconda3/envs/P_main/lib/python3.7/site-packages (from packaging>=17.0->marshmallow->housekeeper==4.2.0) (2.4.7)
Already on 'master'
Already up-to-date.
/home/proj/production/bin/git/housekeeper
INFO  [alembic.runtime.migration] Context impl MySQLImpl.
INFO  [alembic.runtime.migration] Will assume non-transactional DDL.
INFO  [alembic.runtime.migration] Running upgrade 9334201fec74 -> f1669dd42064, Creates a new table describing archiving and retrieval tasks
repository is clean
Logging deploy ...
Getting deployer... done.
Getting last commit message and SHA... done.
Getting version of deploy scripts... /home/proj/production/bin/git/housekeeper
done.
Log deploy... done.
housekeeper, version 4.2.0
```

**Pull request merged**